src/components/NavItem.tsx

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable react/prefer-stateless-function */
```

Comment:
slightly sus 🤔

Reply:
mb on this one, was testing something and forgot to remove the disable